artifactory/usage/reportusage.go

```diff
@@ -67,8 +66,16 @@ func isVersionCompatible(artifactoryVersion string) bool {
 	return version.AtLeast(minArtifactoryVersion)
 }
 
-func reportUsageToJson(productId, commandName string) ([]byte, error) {
+func reportUsageToJson(productId, commandName, serviceId, serverSize string) ([]byte, error) {
 	featureInfo := feature{FeatureId: commandName}
+	if serviceId != "" && serverSize != "" {
+		attributes := map[string]string{
```

Comment:
* The `SendReportUsage` API in jfrog-client-go should remain generic, and not be aware of the specific needs of the `transfer-files` usage report. It therefore should not be aware of attributes like  `serviceId` and `serverSize`. I suggest this API will receieve an optional attributes map.
* I suggest remaning "serverSize" to "sourceStorageSize".

Reply:
Done